qa/tasks/ceph_manager.py

```diff
@@ -1609,17 +1613,12 @@ def run_ceph_w(self, watch_channel=None, shell=False):
                               'cluster', 'audit', ...
         :type watch_channel: str
         """
-        args = ["sudo",
-                "daemon-helper",
-                "kill",
-                "ceph",
-                '--cluster',
-                self.cluster,
-                "-w"]
+        args = self.run_ceph_w_prefix + ['-w']
```

Comment:
You can also try:

`args = ['exec', 'sudo', ...]`

Which will cause the shell to `execve` the `ceph` command, effectively.

Reply:
Tried this change and tested this PR with and without this change during last updates. `run_ceph_w()` still leads to a crash when `shell` is set to `False`, doesn't crash when `shell` is set to `True` and prepending `exec sudo` to command arguments also prevents the crash. I've merged this fix in respective commits on this PR.

If this PR looks okay to you, we can move to final QA testing IMO.

Reply:
Links to last few runs of Ceph API CI job for this PR -

Tested the fix(adding `exec sudo` and setting shell to False) to command arguments; job passed - https://jenkins.ceph.com/job/ceph-api/19955/
Retested if `shell=True` also worked; job passed - https://jenkins.ceph.com/job/ceph-api/19968/
Retested that `shell=False` still leads to crash; job failed (expected) - https://jenkins.ceph.com/job/ceph-api/20024/
Tested the fix (removing `shell=True` and prepending cmd args with `exec sudo`)  merged in respective commit; job passed - https://jenkins.ceph.com/job/ceph-api/20089/

Reply:
Not sure why Ceph API job was re-triggered but it failed - https://jenkins.ceph.com/job/ceph-api/20110/

Reply:
Implemented this solution, it works fine now.